app/inpututils.cpp

```diff
@@ -1129,7 +1129,7 @@ const QUrl InputUtils::getFormEditorType( const QString &widgetNameIn, const QVa
     }
 
     // Mind this hack - fields with `no-gallery-use` won't use gallery, but normal word tags instead
-    if ( field.name().contains( "nogallery", Qt::CaseInsensitive ) || field.alias().contains( "nogallery", Qt::CaseInsensitive ) )
+    if ( fieldName.contains( "nogallery", Qt::CaseInsensitive ) || field.alias().contains( "nogallery", Qt::CaseInsensitive ) )
```

Comment:
Do not check `field.alias` as that one is empty too

```suggestion
    if ( fieldName.contains( "nogallery", Qt::CaseInsensitive ) )
```